src/test/resources/features/resolve/uses_and_modules_name_resolve.feature

```diff
@@ -41,10 +123,10 @@ Feature: Resolve use and module name to its declaration
       input:
         "data_file.txt"
     """
-    When I put the caret at NAME
+    When I put the caret at MODULE as
     Then reference should resolve to "MODULE:" in "foo.smk"
 
-  Scenario: Module name refere to module declaration
+  Scenario: Module name refer to module declaration, confused scenario
```

Comment:
What does `confused` mean here? Is it for module declaration with unresolved snakefile path?

Reply:
Not clear what is confusing in this scenario compared to previous scenario. But I'm indeed confused =)

Reply:
Done. Changed to:

`Scenario: Module name resolving results shouldn't contain use sections for which the module name matched name pattern`